### DISCUSSION TITLE
Create .PFX file in addition to .CRT file when creating SSL keys

### DISCUSSION THREAD

**Discussion by fizpawiz** [Chia-Network/chia-blockchain#15726] [Ideas]
.NET apps have a lot of difficulty using the .CRT files Chia currently produces.  It would be really helpful to *also* create .PFX files with the same key when generating the SSL keys.  Thanks for the suggestion, @michaeltaylor3d

**Comment by esaung** [comment]
Thanks Ken, Here's the corresponding productboard entry for internal tracking - https://chia-network.productboard.com/feature-board/4818004-chia-network-roadmap-internal/features/21427405/detail
